ONSdigital/ras-frontstage
# Help url stuff

# What and why?

The help pages needed to be exposed to the EQ payload by using a `survey_ref` and `ru_ref` in the URLs. These variables are, in turn, used to retrieve the business ID and survey short name.

# How to test?

Log into frontstage, click on "help with this survey", and navigate around the various options. They should all work, and you should see a survey ref (i.e. `139` for QBS) and an ru ref (i.e. `49900000001`) in the help page URLs.

# Trello

[Card](https://trello.com/c/cENWYFNf)
